src/catalog/catalog.cpp

```diff
@@ -269,9 +269,14 @@ ResultType Catalog::CreateSchema(const std::string &database_name,
   auto pg_namespace =
       catalog_map_[database_object->GetDatabaseOid()]->GetSchemaCatalog();
   auto schema_object = pg_namespace->GetSchemaObject(schema_name, txn);
-  if (schema_object != nullptr)
-    throw CatalogException("Schema(namespace) " + schema_name +
+  if (schema_object != nullptr) {
+    //if the temporary schema exists
+    if (schema_name.find(TEMP_NAMESPACE_PREFIX) != std::string::npos) {
```

Comment:
Not sure about what this if statement is checking. If the temporary schema exists, should it throw exceptions anyway?